app/resources/cdl/charge_manager.rb

```diff
@@ -10,6 +10,22 @@ def initialize(resource_id:, eligible_item_service:, change_set_persister:)
       @resource_id = resource_id
       @eligible_item_service = eligible_item_service
       @change_set_persister = change_set_persister
+      clean_expired_charges
+    end
+
+    def clean_expired_charges
+      resource_charge_list.charged_items = resource_charge_list.charged_items.map do |item|
```

Comment:
I think you can just do `resource_charge_list.charged_items = resource_charge_list.charged_items.select(&:expired?)`, and then add the `expired?` method to the `ChargedItem` class.